src/Containers/Footnote.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Main, Responsive } from '@Utils/BaseStyles';
+import {
+    flex, position, scroll, transition
+} from '@Utils/Mixins';
+import { useTransition } from '@Utils/Hooks';
+
+export interface FootnoteProps {
+    children?: React.ReactNode;
+    show?: boolean;
+}
+
+export const Footnote: React.FunctionComponent<FootnoteProps> = ({
+    children,
+    show,
+    ...props
+}) => {
+    const [, mount, animation] = useTransition(show);
```

Comment:
You might want to look into this, because i think the delay from true => false is too short for the animation

Reply:
When I tested it, the animation seemed fine 🤷‍♂️